ariadne/core/src/Ariadne/UIConfig.hs

```diff
@@ -97,6 +107,41 @@ deleteRetypeMkMessage :: Format Text (delItem -> Text) -> delItem -> Text
 deleteRetypeMkMessage itemTypeFormat itemType = sformat
     ("Type " % itemTypeFormat % " name to confirm deletion") itemType
 
+rmUnkownKeysHeaderMessage :: Text
+rmUnkownKeysHeaderMessage = "Remove unknown keys"
+
+rmUnknownKeysSureMessage :: Text
+rmUnknownKeysSureMessage = "Make sure that you don't need keys from keyfile \
+    \(It is not a backup, for example). Those keys will be deleted."
+
+rmUnknownKeysIntroMessage :: Text
+rmUnknownKeysIntroMessage= "There are keys, that don't \
+    \correspond to any known wallet.\nDo you want to delete them?"
+
+rmUnknownKeysRetypeMkMessage :: Text -> Text
+rmUnknownKeysRetypeMkMessage typeMsg = "Please, write " <> typeMsg <> " if you are sure."
+
+rmUnknownKeysRetypeConfirm :: Text
+rmUnknownKeysRetypeConfirm = "Yes"
```

Comment:
In TUI it prints "...type Yes to...", I don't like the formatting. Maybe it should say "... type "Yes" to...", or "Yes" should be in bold or italic.

Reply:
done, will push soon.